src/site/content/en/blog/script-component/index.md

```diff
@@ -0,0 +1,221 @@
+
+## Measuring the impact
+
+We used the templates for the Next.js [commerce app](https://nextjs.org/commerce) and [starter blog](https://github.com/vercel/next-learn/tree/dependabot/npm_and_yarn/seo/demo/next-12.1.0/basics/learn-starter) to create two demo apps that helped measure the impact of including third-party scripts. Commonly used third-parties for Google Tag Manager and social media embeds were included on the pages of these apps directly at first and then through the Script component. We then compared the performance of these pages on [WebPageTest](https://www.webpagetest.org/).
+
+
+
+### Third-party scripts in a Next.js commerce app
+
+Third-party scripts were added to the commerce app template for the demo as given below. 
+
+<div class="table-wrapper scrollbar">
+<table>
+    <thead>
+        <tr>
+            <th>Before</th>
+            <th>After</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>Google Tag Manager with async</td>
+            <td rowspan="2" >Script component with strategy = afterInteractive for both scripts
+            </td>
+        </tr>
+        <tr>
+            <td>Twitter Follow button without async or defer
+            </td>
+        </tr>
+    </tbody>
+    <caption>Script and Script Component configuration for demo 1 with 2 scripts.</caption>
+</table>
+</div>
+
+
+
+The following comparison shows the visual progress for both versions of the [Next.js commerce starter-kit](https://demo.vercel.store/). As seen, LCP occurs nearly 1s earlier with the Script component enabled with the right loading strategy.
+
+{% Img src="image/IypihH3o5cSpEMVp5i08dp69otp2/a3gpB5Ji7vcvQjBhviCP.png", alt="Film strip comparison showing improvment in LCP", width="800", height="372" %}
+
+### Third-party scripts in a Next.js blog
+
+Third-party scripts were added to the demo blog app as given below. 
+
+<div>
+<table>
+    <thead>
+        <tr>
+            <th>Before</th>
+            <th>After</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>Google Tag Manager with async</td>
+            <td rowspan="2" >Script component with strategy = lazyonload for each of the four scripts
+            </td>
+        </tr>
+        <tr>
+            <td>Twitter Follow button with async
+            </td>
+        </tr>
+        <tr>
+            <td>YouTube Subscribe button without async or defer
+            </td>
+        </tr>
+        <tr>
+            <td>LinkedIn Follow button without async or defer
+            </td>
+        </tr>
+    </tbody>
+    <caption>Script and Script Component configuration for demo 2 with 4 scripts.</caption>
+</table>
+</div>
+
+{% Img src="image/IypihH3o5cSpEMVp5i08dp69otp2/BAWQVlyBX1UUJOMyrBJm.gif", alt="Video showing loading progress with and without the Script component ", width="800", height="306" %}
```

Comment:
```suggestion
<figure>
  {% Img src="image/IypihH3o5cSpEMVp5i08dp69otp2/BAWQVlyBX1UUJOMyrBJm.gif", alt="Video showing loading progress with and without the Script component ", width="800", height="306" %}
</figure>
```

Reply:
You might want to expand on the alt text a bit to include before and after metrics so that those on screen readers will get something a bit more specific.